packages/hackmode/lib/index.js

```diff
@@ -1,12 +1,9 @@
 const msgpack = require('msgpack-lite')
 
-const { AsyncClass } = require('@holochain/n3h-common')
-
-const { P2p, Connection } = require('@holochain/n3h-mod-spec')
+const { P2p } = require('@holochain/n3h-mod-spec')
 const { P2pBackendHackmodePeer } = require('./p2p-backend-hackmode-peer')
-const { ConnectionBackendWss } = require('@holochain/n3h-mod-connection-wss')
 
-const config = require('./config')
+const { N3hMode } = require('../../n3h-ipc/lib/n3hMode')
```

Comment:
heh, direct ref to another package, but yeah, if we're going to reorg, not a big deal

Reply:
🤦‍♂️